src/developer-tools/get-started/set-up.html

```diff
@@ -10,12 +10,12 @@
         <div class="editable">
           <h1>Get Started With Interledger Accounts</h1>
           <p>Make payments on a testnet.</p>
-          <p>To get started using Interledger, we'll walk through using a test network, which uses fake money, rather than transferring real value. This Interledger testnet is currently maintained by teams at&nbsp;<a target="_blank" href="https://xpring.io/">Xpring&nbsp;<em></em></a>&nbsp;and&nbsp;<a target="_blank" href="https://coil.com/">Coil</a>, but anyone can join.</p>
+          <p>To get started using Interledger, we'll walk through using a test network, which uses fake money, rather than transferring real value. This Interledger testnet is currently maintained by teams at&nbsp;<a target="_blank" href="https://ripple.com/ripplex/">RippleX&nbsp;<em></em></a>&nbsp;and&nbsp;<a target="_blank" href="https://coil.com/">Coil</a>, but anyone can join.</p>
           <p>The RippleX and Rafiki testnet connectors, both used in this tutorial, together form a network for users to practice sending and receiving test currency values.</p>
           <p>With Interledger, all currency pairs that senders and receivers use are supported. For simplicity, this tutorial demonstrates sending and receiving test XRP only, but in practice senders and receivers can use different currencies.</p>
           <p>This tutorial describes two ways to create an Interledger account to send and receive test funds:</p>
           <ul>
-            <li>With a&nbsp;<a target="_blank" href="https://ripplex.io/">ripplex.io&nbsp;<em></em></a>&nbsp;account</li>
+            <li>With a&nbsp;<a target="_blank" href="https://ripplex.io/">RippleX&nbsp;<em></em></a>&nbsp;account</li>
```

Comment:
ripplex.io says it's been sunset at the end of July. Should this link somewhere else?

```
We've Sunset Ripplex.io

Thank you for visiting ripplex.io!
As of July 31, 2021, this website is no longer being supported.
```

Reply:
Good catch, I will update that

Reply:
This should be updated now.